tests/Sdk/Acceptance/Extension/Tasks/ESLintTaskCest.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Acceptance\Extension\Tasks;
+
+use PHPUnit\Framework\Assert;
+use SprykerSdk\Sdk\Tests\AcceptanceTester;
+
+class ESLintTaskCest
+{
+    /**
+     * @var string
+     */
+    private const COMMAND = 'validation:frontend:eslint';
+
+    /**
+     * @var string
+     */
+    private const CONFIG_FILE = 'node_modules/@spryker/frontend-config.eslint/.eslintrc.js';
+
+    /**
+     * @param \SprykerSdk\Sdk\Tests\AcceptanceTester $I
+     *
+     * @return void
+     */
+    public function testFilesDontContainViolations(AcceptanceTester $I): void
+    {
+        // Arrange
+        $I->cleanReports();
+
+        // Act
+        $process = $I->runSdkCommand([
```

Comment:
What behavior do we expect after a run of `validation:frontend:eslint` without any options?

Reply:
I mean where is the test with simple run like
```$process = $I->runSdkCommand([static::COMMAND]);``` ?

Reply:
By default it looks for the config file, but it can't find it from tests, so there will be an error.
These two is how it should work without any parameters.
testFilesDontContainViolations
testFilesContainViolations

Reply:
Ok